src/Fabulous.XamarinForms/Views/Controls/Button.fs

```diff
@@ -47,9 +104,120 @@ type ButtonModifiers =
         )
 
     [<Extension>]
-    static member inline font(this: WidgetBuilder<'msg, #IButton>, value: double) =
+    static member inline textTransform(this: WidgetBuilder<'msg, #IButton>, value: TextTransform) =
+        this.AddScalar(Button.TextTransform.WithValue(value))
+
+    [<Extension>]
+    static member inline cornerRadius(this: WidgetBuilder<'msg, #IButton>, value: int) =
+        this.AddScalar(Button.CornerRadius.WithValue(value))
+
+    [<Extension>]
+    static member inline borderColor(this: WidgetBuilder<'msg, #IButton>, light: Color, ?dark: Color) =
+        this.AddScalar(
+            Button.BorderColor.WithValue(
+                { Light = light
```

Comment:
Maybe we should write a little helper function (inlined) taking care have creating the `AppThemeValue<T>` instead of writing it each time

Reply:
I have added a inline member on the AppThemeValue to make easier